### DISCUSSION TITLE
Can I use Wikis on a private repos of an organization that I (a github PRO member) own?

### DISCUSSION THREAD

**Discussion by malandra-iot** [community/community#32458] [New to GitHub]
Hi all,

This may sound like a very silly question but it is driving me crazy and cannot find a way to make it work.

So, I have a PRO account and I should be able to use WIKIs in private repos as well. 
I have created an organization with the lab I manage. I created some projects under the organization (I appear as "owner" of this organization. However, when I access the homepage of these projects, I do not see the option to create WIKIs in private repos, which should be allowed as I, the owner, have a PRO account. Anyone has an idea why I do not see and knows a way to make it work, if any?

Thanks,
Filippo

**Comment by queenofcorgis** [comment]
Hi there! 

Have you checked the settings to make sure that the Wiki feature is enabled? 

Cheers, 
Lili

**Reply by malandra-iot** [reply]
Hi Lili,

It asks to "upgrade" (see below), but I should not be required to upgrade since I already have an upgraded plan as I am a PRO member.
![image](https://user-images.githubusercontent.com/70348807/191329100-f92181c7-e455-438b-8c6a-255243ea89be.png)

Thanks,
Filippo

**Reply by queenofcorgis** [reply]
I'd recommend reaching out to our [Support team](https://support.github.com/) to see if they can get to the bottom of this!

**Comment by malandra-iot** [comment]
Hi Lili,

It asks to "upgrade" (see below), but I should not be required to upgrade since I already have an upgraded plan as I am a PRO member.

Thanks,
Filippo
[cid:ca596d94-f836-4b4a-9b5c-f4ddc5946fcb]


[cid:8e571d7a-a256-4624-9268-c88d93ac4a65]

Filippo Malandra, Ph.D.
Assistant Professor
Electrical Engineering Department, University at Buffalo
222 Davis Hall, Buffalo, NY 14260
Phone: (716) 645-1151, Email: ***@***.***
​





________________________________
